ibis/backends/dask/execution/window.py

```diff
@@ -49,11 +184,43 @@ def _post_process_empty(
         return series[0]
     else:
         # Project any non delayed object to the shape of "parent"
-        return parent.apply(lambda row: result, meta=(None, 'object'))
+        if isinstance(parent, dd.DataFrame):
+            parent = parent[parent.columns[0]]
+
+        return parent.apply(lambda row, result=result: result, meta=(None, 'object'))
+
+
+def _post_process_order_by(
+    series,
+    parent: dd.DataFrame,
+    order_by: list[str],
+    group_by: list[str],
+    timecontext: TimeContext | None,
+    **kwargs,
+) -> dd.Series:
+    """Functions like pandas with dasky argsorting."""
+    assert order_by and not group_by
+    # FIXME This is likely not needed anymore.
+    # _ibis_index should take care of order_by
+    return series
 
 
-@execute_node.register(ops.Window, dd.Series)
-def execute_window_frame(
+def _post_process_group_by(
+    series,
+    parent: dd.DataFrame,
+    order_by: list[str],
+    group_by: list[str],
+    timecontext: TimeContext | None,
+    op,
+    **kwargs,
+) -> dd.Series:
+    assert not order_by and group_by
+    # FIXME This is likely not needed anymore.
```

Comment:
Should we just not have these?

Reply:
Yeah, I mostly left this for when I do cleanup work in here later.
How we handle `_get_post_process_function` method needs to be refactored.